### DISCUSSION TITLE
Cannot open application

### DISCUSSION THREAD

**Discussion by JessicaWang2** [PlayCover/PlayCover#544] [Q&A]
<img width="257" alt="Screenshot 2022-11-27 at 10 36 51 AM" src="https://user-images.githubusercontent.com/38387162/204153453-b2b12ee8-2453-4a6f-a096-8c46ab494631.png">
<img width="659" alt="Screenshot 2022-11-27 at 10 38 02 AM" src="https://user-images.githubusercontent.com/38387162/204153478-2ac6cac0-c2ca-4cec-b85d-6db0279851e9.png">
Is there something that I missed?

**Comment by amirsaam** [comment]
Apparently you are trying to install PlayCover on an Intel Mac that are not supported.
PlayCover only works on Apple Silicon Macs (e.g. M1 and M2 SOC family)
